accession-commons-core/src/main/java/uk/ac/ebi/ampt2d/commons/accession/core/AccessioningService.java

```diff
@@ -27,18 +27,39 @@
 
 import java.util.List;
 
+/**
+ * Service for retrieval and modifications of object accessions.
```

Comment:
It also provides creation of accessions.

Reply:
Correct, even i had commented same https://github.com/EBIvariation/accession-commons/pull/55#discussion_r266867367